Test/Int/DvdTimeout.expected

```diff
@@ -1,18 +1,5 @@
-goal: dvd a (b + c + d) = true
+Test/Int/DvdTimeout.lean:13:4: error: something went wrong.
+stdout:
 
-query:
```

Comment:
This is probably a regression right? We should print the query before the running `StateM.run` in case the solving fails as in this case.